docs/_articles/en/soql/writing.md

```diff
@@ -17,7 +17,6 @@ VS Code supports writing SOQL in both Apex files and standalone `.soql` files. W
 
 VS Code supports code completions for SOQL `.soql` files (and will eventually support code completion in Apex files). To use this feature, you must refresh the SObject definitions so that the SOQL language server can provide code completion suggestions. Run `SFDX: Refresh SObject Definitions` from the Command Palette.
```

Comment:
```suggestion
VS Code supports code completions for SOQL `.soql` files (and plan to eventually support code completion in Apex files). To use this feature, refresh the SObject definitions so that the SOQL language server can provide code completion suggestions. Run `SFDX: Refresh SObject Definitions` from the Command Palette.
```

Reply:
@smaddox-sf OK to make this update?

Reply:
@sbudhirajadoc - See comment above.  We already support code completion for embedded Apex.